src/node/README.md

```
@@ -18,7 +18,7 @@ $ conan config install https://github.com/k-nuth/ci-utils/raw/master/conan/confi
 2. Install the appropriate library:
 
 ```
-$ conan install --requires=node/0.81.1 --update
+$ conan install --requires=node/0.82.0 --update
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)